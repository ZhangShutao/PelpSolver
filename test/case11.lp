p :- K[0.4, 0.6] p.