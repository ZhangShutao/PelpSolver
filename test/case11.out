WorldView 1:
{[1.0]:{}}